{"uuid":"ec497066-f011-4914-8e10-9f5179b36516","historyId":"b6cba21b2f7e9f26c65643071eab470","fullName":"okhttp.LoginTests.registrationNegativeTest_ExRes400","labels":[{"name":"package","value":"okhttp.LoginTests"},{"name":"testClass","value":"okhttp.LoginTests"},{"name":"testMethod","value":"registrationNegativeTest_ExRes400"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"QA_44_PhoneBook_Roman_Sich"},{"name":"subSuite","value":"okhttp.LoginTests"},{"name":"host","value":"DESKTOP-716S01A"},{"name":"thread","value":"25396@DESKTOP-716S01A.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"registrationNegativeTest_ExRes400","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"The following asserts failed:\n\tresponse code expected [400] but found [401],\n\tstatus expected [400] but found [401],\n\terror expected [Bad request] but found [Unauthorized],\n\tmessage valid expected [true] but found [false]","trace":"java.lang.AssertionError: The following asserts failed:\n\tresponse code expected [400] but found [401],\n\tstatus expected [400] but found [401],\n\terror expected [Bad request] but found [Unauthorized],\n\tmessage valid expected [true] but found [false]\r\n\tat org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)\r\n\tat org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)\r\n\tat okhttp.LoginTests.registrationNegativeTest_ExRes400(LoginTests.java:87)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:568)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:136)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:658)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:219)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:923)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:192)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:808)\r\n\tat org.testng.TestRunner.run(TestRunner.java:603)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:429)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:326)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1169)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1092)\r\n\tat org.testng.TestNG.run(TestNG.java:1060)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1730202287379,"stop":1730202293528}